Common/Data/UniverseSelection/Universe.cs

```diff
@@ -206,6 +206,17 @@ public virtual Security CreateSecurity(Symbol symbol, IAlgorithm algorithm, Mark
                 UniverseSettings.ExtendedMarketHours, false, false, algorithm.LiveMode, symbol.ID.SecurityType == SecurityType.Option);
         }
 
+        /// <summary>
+        /// Returns an existing security for this universe
+        /// </summary>
+        public virtual Security GetSecurity(Symbol symbol, IAlgorithm algorithm)
+        {
+            // by default return the security from the algorithm security collection
+            Security security;
+            algorithm.Securities.TryGetValue(symbol, out security);
```

Comment:
This wasn't initially how I saw this. I was originally thinking `GetSecurity` would *always* return a non-null security, creating if necessary. Setting it up like this does absolutely work but also provides an opportunity for a discontinuity between `GetSecurity` and `CreateSecurity` -- for example, `GetSecurity` could feasibly return a different security than the one that was created using `CreateSecurity` -- I'll ping you in slack to hash this one out -- I'm think it'll be good as is but want to cover our bases.

Reply:
I did it this way to ensure the existing behavior in `UniverseSelection.ApplyUniverseSelection` is unchanged, just by moving the `TryGetValue` call into `Universe.GetSecurity` -- at least for this PR didn't want to change too much :)